public/setup.php

```diff
@@ -620,24 +631,20 @@ function do_software_environment_check()
         $error[] = "There is no database (PDO) support in your PHP setup, you MUST install a suitable PHP PDO extension (e.g. pdo_pgsql, pdo_mysql or pdo_sqlite).";
     }
 
-    if ($m_pdo_mysql) {
-        $info[] = "Database - MySQL support available";
-    } else {
-        $info[] = "Database - MySQL (pdo_mysql) extension not found";
+    if (!$m_pdo_mysql && $CONF['database_type'] == 'mysql') {
+        $error[] = "Database support - MySQL compatible extension (pdo_pgsql) not found";
```

Comment:
That error should probably be `pdo_mysql` ;-) - and the database type for this case could also be `mysqli`